clients/drcachesim/simulator/cache_fifo.h

```diff
@@ -54,12 +54,11 @@ class cache_fifo_t : public cache_t {
     {
```

Comment:
> ix: Small Fixes - Consistent Naming, Error Printing, and add_parent. 

Please give some idea of what component we're talking about: "Small fixes in drcachesim: ..." for example.   "Cleanup" might be better: and see below where add_parent should probably be removed.  So "drcachesim cleanup: parameter names + TLB error messages and destruction" or something.